### PR TITLE
Video Block Placeholder Issue

### PR DESCRIPTION
**Pull request by Mark-M-Ensemble**
When selecting play on a video block with picture, the placeholder image is removed and a video of different height is added. This creates a jump as the HTML is removed an added.

Fix : https://pgatour.atlassian.net/browse/AEMFDEV-89

Test URLs:
- Before: https://main--pgatour-com--hlxsites.hlx.page/pages/whats-new
- After: https://aemfdev-89--video-block-issue--pgatour-com--hlxsites.hlx.page/pages/whats-new

### CODE REVIEW

File: pages/blocks/video/video.js [hunk 1]
```diff
@@ -31,7 +32,10 @@ export default async function decorate(block) {
       wrapper.innerHTML = '<div class="video-placeholder-play"><button title="Play"></button></div>';
       wrapper.prepend(pic);
       wrapper.addEventListener('click', () => {
-        renderVideo(block, source, true);
+        requestAnimationFrame(() => {
```
Comment (shsteimer) [inline pages/blocks/video/video.js:35]: curious about the use of `requestAnimationFrame` to be honest, I've never used it, so if it's required then fair enough, but reading what it does at https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame it doesn't seem to me like it should be required in this case (but what do I know)

File: pages/blocks/video/video.js [hunk 2]
```diff
@@ -31,7 +32,10 @@ export default async function decorate(block) {
...
+          block.setAttribute('style', `height:${wrapper.offsetHeight}px`);
```
Comment (shsteimer) [inline pages/blocks/video/video.js:36]: I think you may also need to set the block as position relative here, and in the css, make the `video.centered` element position absolute. at least for me, the centering doesn't appear to work as it is currently.

File: pages/blocks/video/video.js (hunk 1 above)
Comment (Mark-M-Ensemble) [reply pages/blocks/video/video.js:35]: Thanks I've removed... I had left that in from first solution

File: pages/blocks/video/video.js (hunk 2 above)
Comment (Mark-M-Ensemble) [reply pages/blocks/video/video.js:36]: Sorry what is it you're seeing, not vertically or horizontally centered?

Comment (Mark-M-Ensemble) [reply pages/blocks/video/video.js:36]: 538A8E9D-AE0E-43D8-8EB9-6CD807A168E5

Comment (shsteimer) [reply pages/blocks/video/video.js:36]: not vertically centered, this on desktop safari.

before clicking play
image


after clicking play

image

Comment (Mark-M-Ensemble) [reply pages/blocks/video/video.js]: Thanks, hadn't tested Safari.

This has been resolved.

**Comment by Mark-M-Ensemble** [comment]
@shsteimer This has been updated to fix the issues raised by your comments. Should be ready to go if it looks good on your end.

**Review by fkakatie — APPROVED** [review]

**Pull request merged**